mpqp/tools/generics.py

```diff
@@ -153,6 +154,51 @@ def find_index(iterable: Iterable[T], oracle: Callable[[T], bool]) -> int:
     raise ValueError("No objects satisfies the given oracle")
 
 
+def compute_expected_matrix(qcircuit: QCircuit):
```

Comment:
Shouldn't we put the input type `"QCircuit"` as a string here? (since we only import it in the if TYPE_CHECKING)

Reply:
done